src/components/Books.js

```diff
@@ -0,0 +1,54 @@
+const Books = () => {
+  const books = [
+    {
+      id: 1,
+      title: 'The Road to React',
+      author: 'Robin Wieruch',
+    },
+    {
+      id: 2,
+      title: 'Learning Node',
+      author: ' Shelley Powers',
+    },
+    {
+      id: 3,
+      title: 'Node.js',
+      author: 'Sebastian Springer',
+    },
+    {
+      id: 4,
+      title: 'Javascript for Web Developer',
+      author: 'Matt Frisbie',
+    },
+  ];
+  return (
+    <>
+      <div className="">
+        { books.map((book) => (
+          <p key={book.id}>
+            {book.title}
+            {' '}
+            written by
+            {' '}
+            {book.author}
+          </p>
+        )) }
+      </div>
+      <hr />
+      <div className="">
+        <h2>Add new book</h2>
+        <form>
+          <input type="text" placeholder="Book title" />
+          <select name="catagory" id="category">
+            <option value="NodeJS">NodeJS</option>
+            <option value="React">React</option>
+            <option value="Javascript">Javascript</option>
+          </select>
```

Comment:
- [ ] According to the project requirements, please replace the select input for "category" with an input for "author"

![image](https://github.com/joramot/bookstore_react/assets/46839548/44e15059-f103-4f5b-b35c-ae5c7a3461a8)
![image](https://github.com/joramot/bookstore_react/assets/46839548/bc10e30a-f9c3-4dc5-925e-7f3324a78eec)

Reply:
Addressed